packages/discovery-react-components/src/components/StructuredQuery/__stories__/default.md

```diff
@@ -0,0 +1,3 @@
+#### Overview
+
+The `StructuredQuery` component will provide the ability to visually build structured queries against your project. Full functionality will be added in the future.
```

Comment:
probably don't need to say anything about the future

Reply:
Okay, either way makes sense to me. So, just to confirm on the same page, you're thinking no overview at all for now and we add it in later?